news/main.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* <news item>
+
```

Comment:
Here you can mention the issue that is fixed. That way, it will appear in the release notes.

Reply:
Sure. and Sir,  2 checks are failing(macos, ubuntu). Is that OK?

Reply:
If the failing tests are not relevant to this change, I will rerun it.